tests/fv3_conf/cpld_datm_gefs.IN

```diff
@@ -22,4 +22,6 @@ if [[ $OCNRES == '025' ]]; then
 else
   cp    @[INPUTDATA_ROOT]/MOM6_IC/@[OCNRES]/2011100100/MOM6_IC_TS*.nc ./INPUT/MOM6_IC_TS.nc
   cp    @[INPUTDATA_ROOT]/CICE_IC/@[OCNRES]/cice_model_@[ICERES].cpc*.nc cice_model.res.nc
+  # MOM6 increment file
+  cp    @[INPUTDATA_ROOT]/MOM6_IC/@[OCNRES]/2011100100/mom6_increment.nc ./INPUT/mom6_increment.nc
```

Comment:
I am curious, so the increments in this file will be applied to the corresponding MOM6 fields every MOM time step during the forecast time fh=0-MOM_IAU_HRS? How about restart runs, will it be applied from current starting time to starting time +MOM_?IAU_HRS

Reply:
Correct, that is the way I am doing the replay.